src/super_gradients/training/pipelines/pipelines.py

```diff
@@ -286,7 +289,7 @@ def _decode_model_output(self, model_output: Union[List, Tuple, torch.Tensor], m
         return predictions
 
     def _instantiate_image_prediction(self, image: np.ndarray, prediction: DetectionPrediction) -> ImagePrediction:
-        return ImageDetectionPrediction(image=image, prediction=prediction, class_names=self.class_names)
+        return ImageClassificationPrediction(image=image, prediction=prediction, class_names=self.class_names)
```

Comment:
I think that's a mistake

Reply:
@danielafrimi looks like you should undo this change and go back to using `ImageDetectionPrediction` here